apps/nextjs-website/src/components/organisms/WebinarQuestionsTemplate/WebinarQuestionsTemplate.tsx

```diff
@@ -69,24 +72,28 @@ const WebinarQuestionsTemplate = ({
                 </TableRow>
               </TableHead>
               <TableBody>
-                {sortedQuestions.map((row) => (
+                {sortedQuestions.map((webinarQuestion) => (
                   <WebinarQuestionRow
-                    key={row.createdAt.toJSON()}
-                    question={row}
+                    key={webinarQuestion.createdAt.toISOString()}
+                    question={webinarQuestion}
                     userName={userName}
                     onHide={async (hide) =>
-                      await updateQuestion(
-                        row,
-                        'hide',
-                        hide ? userName : undefined
-                      )
+                      await updateWebinarQuestion({
+                        webinarId: webinarQuestion.webinarId,
+                        createdAt: webinarQuestion.createdAt,
+                        hiddenBy: hide
+                          ? { operation: 'update', value: userName }
+                          : { operation: 'remove' },
+                      })
                     }
                     onHighlight={async (highlight) =>
-                      await updateQuestion(
-                        row,
-                        'highlight',
-                        highlight ? userName : undefined
-                      )
+                      await updateWebinarQuestion({
+                        webinarId: webinarQuestion.webinarId,
+                        createdAt: webinarQuestion.createdAt,
+                        highlightedBy: highlight
```

Comment:
Same as the comment before, the name it's not super clear given the object passed